capabilities/src/execute_task_solution_capability.cpp

```diff
@@ -168,14 +168,9 @@ bool ExecuteTaskSolutionCapability::constructMotionPlan(const moveit_task_constr
 				RCLCPP_DEBUG(LOGGER, "Using JointModelGroup '%s' for execution", group->getName().c_str());
 			}
 		}
-		exec_traj.trajectory = std::make_shared<robot_trajectory::RobotTrajectory>(model, group);
-		exec_traj.trajectory->setRobotTrajectoryMsg(state, sub_traj.trajectory);
-
-		if (!sub_traj.controller_names.empty()) {
-			for (auto cn : sub_traj.controller_names) {
-				exec_traj.controller_names_.push_back(cn.data.data());
-			}
-		}
+		exec_traj.trajectory_ = std::make_shared<robot_trajectory::RobotTrajectory>(model, group);
+		exec_traj.trajectory_->setRobotTrajectoryMsg(state, sub_traj.trajectory);
```

Comment:
The names (`_`) are changed back and forth. Please further cleanup history to have minimal changes per commit. 
This also facilitates reviewing :wink: